Samples/Hast.Samples.SampleAssembly/ImageSharpModifications/Resize/ResizeExtensions.cs

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+// Modified file, original is found at:
+// https://github.com/SixLabors/ImageSharp/blob/master/src/ImageSharp/Processing/Extensions/Transforms/ResizeExtensions.cs
+
+using Hast.Layer;
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Processing;
+using SixLabors.ImageSharp.Processing.Processors.Transforms;
+using System;
+
+namespace Hast.Samples.SampleAssembly.ImageSharpModifications.Resize;
+
+public static class ResizeExtensions
+{
+    /// <summary>
+    /// Resizes an image to the given width and height.
+    /// </summary>
+    /// <param name="source">The image to resize.</param>
+    /// <param name="width">The target image width.</param>
+    /// <param name="height">The target image height.</param>
+    /// <returns>The <see cref="IImageProcessingContext"/> to allow chaining of operations.</returns>
+    /// <remarks><para>Passing zero for one of height or width will automatically preserve
+    /// the aspect ratio of the original image or the nearest possible ratio.</para></remarks>
+    public static IImageProcessingContext HastResize(
+        this IImageProcessingContext source,
+        int width,
+        int height)
+        => HastResize(
```

Comment:
Could you move all the arrows in the file to the previous lines? (after the closing parenthesis)